src/model.js

```diff
@@ -3513,16 +3509,27 @@ Instead of specifying a Model, either:
    * @returns {object}
    */
   where(checkVersion) {
+    if (this.constructor.primaryKeyAttributes.length === 0) {
+      throw new Error(
+        `This model instance method needs to be able to identify the entity in a stable way, but the model does not have a primary key attribute definition. Either add a primary key to this model, or use one of the following alternatives:
+  
+      - instance methods "save", "update", "decrement", "increment": Use the static "update" method instead.
```

Comment:
Can you de-indent this? The log will include the extra whitespace at the beginning of this line